CMakeLists.txt

```diff
@@ -23,7 +23,13 @@ add_library( DemoGafferExtensionModule SHARED ${DemoGafferExtensionModuleSrc} )
 set_target_properties( DemoGafferExtensionModule PROPERTIES PREFIX "" OUTPUT_NAME "_DemoGafferExtension" )
 target_compile_definitions( DemoGafferExtensionModule PRIVATE BOOST_SIGNALS_NO_DEPRECATION_WARNING=1 )
 target_link_libraries( DemoGafferExtensionModule DemoGafferExtension )
-target_include_directories( DemoGafferExtensionModule PRIVATE include ${GAFFER_ROOT}/include ${GAFFER_ROOT}/lib/Python.framework/Versions/2.7/include/python2.7 )
+
+if(DARWIN)
+  target_include_directories( DemoGafferExtensionModule PRIVATE include ${GAFFER_ROOT}/include ${GAFFER_ROOT}/lib/Python.framework/Versions/2.7/include/python2.7 )
+else()
+  target_include_directories( DemoGafferExtensionModule PRIVATE include ${GAFFER_ROOT}/include ${GAFFER_ROOT}/include/python2.7 )
+endif(DARWIN)
```

Comment:
I'm not super familiar with CMake, but is this the right syntax for closing out an `if`? I'm just surprised to see we need to re-specify the initial condition...

Reply:
Not mandatory at all but it can help on concatenated statements. Mainly for clarity sake.